Assets/Kottaboas/Scripts/GameManager.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//Controls the game of Kottabos
+public class GameManager : MonoBehaviour
```

Comment:
either put in a namespace or rename to something less generic like ````KottaboasGameManager```` we've got a lot of things in the game that could be considered a "Game" :)